<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>719</width>
    <height>614</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="mainHorLayout">
      <item>
       <widget class="QFrame" name="leftFrame">
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QVBoxLayout" name="leftVerLayout">
           <item>
            <layout class="QHBoxLayout" name="leftToolLayout">
             <item>
              <widget class="QLabel" name="mapToolLabel">
               <property name="text">
                <string>Keyword maps</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="Line" name="line_7">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="deleteMapBtn">
               <property name="text">
                <string>delete</string>
               </property>
               <property name="popupMode">
                <enum>QToolButton::DelayedPopup</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="Line" name="line_8">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="renameMapBtn">
               <property name="text">
                <string>rename</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="Line" name="line_9">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QCommandLinkButton" name="addMapBtn">
             <property name="text">
              <string>Add new map</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="leftVerSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="rightFrame">
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="rightVerLayout">
           <item>
            <layout class="QHBoxLayout" name="rightToolLayout">
             <item>
              <widget class="QLabel" name="rightToolLabel">
               <property name="text">
                <string>Keywords</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="Line" name="line">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="deleteWordBtn">
               <property name="text">
                <string>delete</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="Line" name="line_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="renameWordBtn">
               <property name="text">
                <string>rename</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="Line" name="line_1">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QCommandLinkButton" name="addWordBtn">
             <property name="text">
              <string>Add new word</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="rightVerSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
